data_explorer/app/main.py

```diff
@@ -18,6 +19,15 @@
 
 if __name__ == "__main__":
     # make sidebar with input fields for manifest path, subset and fields
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--base_path",
+        type=str,
+        help='Mounted or remote base path',
+    )
+    args = parser.parse_args()
+
+    # TODO: add remote path as an argument (Later PR)
```

Comment:
I think it would be nice if you could create an issue for this task, so that we don't forget about fixing it later on.